discord/scheduled_events.py

```diff
@@ -230,23 +626,70 @@ def __repr__(self) -> str:
             f"<ScheduledEvent id={self.id} "
             f"name={self.name} "
             f"description={self.description} "
-            f"start_time={self.start_time} "
-            f"end_time={self.end_time} "
+            f"start_time={self.scheduled_start_time} "
+            f"end_time={self.scheduled_end_time} "
             f"location={self.location!r} "
             f"status={self.status.name} "
-            f"subscriber_count={self.subscriber_count} "
+            f"user_count={self.user_count} "
             f"creator_id={self.creator_id}>"
+            f"channel_id={self.channel_id}>"
         )
 
+    @property
+    @deprecated(instead="entity_metadata.location", since="2.7", removed="3.0")
+    def location(self) -> ScheduledEventLocation | None:
+        """Returns the location of the event."""
+        if self.channel_id is None:
+            self.location = ScheduledEventLocation(
+                state=self._state, value=self.entity_metadata.location
+            )
+        else:
+            self.location = ScheduledEventLocation(
+                state=self._state, value=self.channel_id
+            )
+
     @property
     def created_at(self) -> datetime.datetime:
         """Returns the scheduled event's creation time in UTC."""
         return utils.snowflake_time(self.id)
 
+    @property
+    @deprecated(instead="scheduled_start_time", since="2.7", removed="3.0")
+    def start_time(self) -> datetime.datetime:
+        """
+        Returns the scheduled start time of the event.
+
+        .. deprecated:: 2.7
+            Use :attr:`scheduled_start_time` instead.
+        """
+        return self.scheduled_start_time
+
+    @property
+    @deprecated(instead="scheduled_end_time", since="2.7", removed="3.0")
+    def end_time(self) -> datetime.datetime | None:
+        """
+        Returns the scheduled end time of the event.
+
+        .. deprecated:: 2.7
+            Use :attr:`scheduled_end_time` instead.
+        """
+        return self.scheduled_end_time
+
+    @property
+    @deprecated(instead="user_count", since="2.7", removed="3.0")
+    def subscriber_count(self) -> int | None:
+        """
+        Returns the number of users subscribed to the event.
+
+        .. deprecated:: 2.7
+            Use :attr:`user_count` instead.
+        """
+        return self.user_count
```

Comment:
I don't really see the point of renaming these, sure it's not the API official name, but it's not that bad *here* I feel like.

Reply:
Yeah I agree for subscriber count that user_count was also confusing naming, but idk about the other two